app.js

```diff
@@ -1,5 +1,36 @@
 'use strict';
 
-const app = require('./');
+const hof = require('hof');
+const config = require('./config');
+const mockAPIs = require('./mock-apis');
+const bodyParser = require('busboy-body-parser');
+const notProd = config.env !== 'production';
+
+const whatCSP = () => {
+  if (notProd) {
+    return false;
+  }
+  return {
+    // required to use data:image for image-preview
+    imgSrc: ['data:']
+  };
+};
+
+const options = {
+  start: false,
+  routes: [
+    require('./apps/rotm')
+  ],
+  csp: whatCSP()
+};
+
+const app = hof(options);
+
+if (notProd) {
```

Comment:
It would be good to have the mounting of mocks derived from an independently configurable point of config rather than directly resolved from `NODE_ENV`. It can default to using `NODE_ENV`, if not configured, but should be independent to allow fine-grained config.